Qt/mainwindow.cpp

```diff
@@ -493,118 +511,87 @@ void MainWindow::createMenus()
-		->addEventChecked(&g_Config.bVertexCache);
-	videoMenu->add(new MenuAction(this, SLOT(frameskipAct()),     QT_TR_NOOP("&Frameskip")))
-		->addEventChecked(&g_Config.iFrameSkip);
-	videoMenu->add(new MenuAction(this, SLOT(frameskipTypeAct()),     QT_TR_NOOP("&FrameSkipType")))
-		->addEventChecked(&g_Config.iFrameSkipType);
-	optionsMenu->add(new MenuAction(this, SLOT(audioAct()),   QT_TR_NOOP("&Audio")))
-		->addEventChecked(&g_Config.bEnableSound);
-	optionsMenu->addSeparator();
+	// Game settings
+	MenuTree* gameSettingsMenu = new MenuTree(this, menuBar(), QT_TR_NOOP("&Game settings"));
+	gameSettingsMenu->add(new MenuAction(this, SLOT(keepOnTopAct()),   QT_TR_NOOP("&Keep PPSSPP on top")))
+		->addEventChecked(&g_Config.bTopMost);
+	gameSettingsMenu->add(new MenuAction(this, SLOT(pauseWhenNotFocusedAct()),   QT_TR_NOOP("&Pause when not focused")))
+		->addEventChecked(&g_Config.bPauseOnLostFocus);
+	gameSettingsMenu->add(new MenuAction(this, SLOT(languageAct()),        QT_TR_NOOP("La&nguage...")));
+	gameSettingsMenu->add(new MenuAction(this, SLOT(controlMappingAct()),        QT_TR_NOOP("C&ontrol mapping...")));
+	gameSettingsMenu->add(new MenuAction(this, SLOT(displayLayoutEditorAct()),        QT_TR_NOOP("Display layout editor...")));
+	gameSettingsMenu->add(new MenuAction(this, SLOT(moreSettingsAct()),        QT_TR_NOOP("&More settings...")));
+	gameSettingsMenu->addSeparator();
 #if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
-	optionsMenu->add(new MenuAction(this, SLOT(fullscrAct()), QT_TR_NOOP("&Fullscreen"), Qt::Key_F11))
+	gameSettingsMenu->add(new MenuAction(this, SLOT(fullscrAct()), QT_TR_NOOP("&Fullscreen"), Qt::Key_F11))
 #else
-	optionsMenu->add(new MenuAction(this, SLOT(fullscrAct()), QT_TR_NOOP("&Fullscreen"), QKeySequence::FullScreen))
+	gameSettingsMenu->add(new MenuAction(this, SLOT(fullscrAct()), QT_TR_NOOP("Fu&llscreen"), QKeySequence::FullScreen))
 #endif
 		->addEventChecked(&g_Config.bFullScreen);
-	optionsMenu->add(new MenuAction(this, SLOT(statsAct()),   QT_TR_NOOP("&Show debug statistics")))
-		->addEventChecked(&g_Config.bShowDebugStats);
-	optionsMenu->add(new MenuAction(this, SLOT(showFPSAct()), QT_TR_NOOP("&Show FPS")))
+	gameSettingsMenu->add(new MenuAction(this, SLOT(VSyncAct()),   QT_TR_NOOP("VS&ync")))
+		->addEventChecked(&g_Config.bVSync);
+	MenuTree* postprocessingShaderMenu = new MenuTree(this, gameSettingsMenu, QT_TR_NOOP("Postprocessin&g shader"));
+	postprocessingShaderGroup = new MenuActionGroup(this, postprocessingShaderMenu, SLOT(postprocessingShaderGroup_triggered(QAction *)),
+		QStringList() << "Off" << "FXAA Antialiasing" << "CRT scanlines" << "Natural Colors" << "Natural Colors (no blur)" << "Vignette" << "Grayscale" << "Bloom" << "Sharpen" << "Inverse Colors" << "Scanlines (CRT)" << "Cartoon" << "4xHQ GLSL" << "AA-Color" << "Spline36 Upscaler" << "5xBR" << "5xBR-lv2" << "Video Smoothing AA" << "Super Sampling AA (Gauss)",
```

Comment:
On Windows, we use `GetAllPostShaderInfo()` which dynamically loads the available shaders.  This list here is hardcoded, and may not match the list of shaders the user actually has.  If we add/remove a shader, no one will remember to update this list - especially because it's automatically updated on Windows.

There's some complexity here with language translations and timing, so it might be another thing to do in a follow-up issue/pull.

-[Unknown]

Reply:
Then I'll remove it, thanks.